steps/create-a-new-release-in-jira/1.0.1/step.yml

```diff
@@ -0,0 +1,98 @@
+title: Create a new release in JIRA
+summary: |
+  Create a new release in JIRA.
+description: |
+  This steps create a Release in your JIRA project, with the start date, publication and description you need.
+website: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+source_code_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+support_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira/issues
+published_at: 2021-10-07T10:40:41.960748-03:00
+source:
+  git: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira.git
+  commit: 33aff55ee19bd1e089c9bf90273f3ee58fed5ff9
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
```

Comment:
NOTE: this is the default toolkit value, you can omit the whole `toolkit` property this case.